whois-api/src/test/java/net/ripe/db/whois/api/log/JettyRequestLogTestIntegration.java

```diff
@@ -101,19 +103,43 @@ public void password_filtered() throws Exception {
                 .get(WhoisResources.class);
 
 
-        String actual = fileToString(getRequestLogFilename());
+        String actual = fileToString(getRequestLog());
         assertThat(actual, containsString("GET /whois/test/person/TP1-TEST?password=FILTERED"));
         assertThat(actual, not(containsString("some-api_key-123")));
     }
 
+    @Test
+    public void password_and_override_filtered() throws Exception {
```

Comment:
Add tests for mixed-case (and/or uppercase) password and override. Query parameters are case-sensitive, but if a user mixed the case by mistake we don't want sensitive data in the log.

Reply:
done in 2bc549bab